packages/oidc-middleware/notes.md

```diff
@@ -0,0 +1 @@
+what should the default session options be?
```

Comment:
Maybe you want to remove the notes.md file?

Reply:
oops, thanks